### PR TITLE
Inspect sent messages

### PR DESCRIPTION
**Pull request by KnorpelSenf**
As not understood by https://t.me/grammyjs/116786

### CODE REVIEW

**Review by roj1512 — APPROVED** [review]
LGTM. Thank you so much!

**Review by CikiMomogi — APPROVED** [review]

File: site/docs/zh/guide/basics.md [hunk 1]
```diff
@@ -41,6 +41,9 @@ Bot 可以使用的所有方法（**[重要的列表](https://core.telegram.org/
 await bot.api.sendMessage(12345, "Hi!");
 // 你也可以选择性地传入一个选项对象。
 await bot.api.sendMessage(12345, "Hi!", {/* 其他选项 */});
+// 检查已发送消息的消息对象
```
Comment (KnorpelSenf) [inline site/docs/zh/guide/basics.md:44]:
```suggestion
// 检查已发送消息的消息对象。
```

Comment (KnorpelSenf) [reply site/docs/zh/guide/basics.md:44]: @roj1512 please apply if you agree

File: site/docs/id/guide/basics.md [hunk 2]
```diff
@@ -39,10 +39,13 @@ Semua method yang dapat digunakan oleh bot (**[daftar penting](https://core.tele
 ```ts
 // Mengirim sebuah pesan ke pengguna 12345.
 await bot.api.sendMessage(12345, "Halo!");
-// Selain itu, kamu bisa menambahkan opsi ke object.
+// Selain itu, kamu juga bisa menambahkan beberapa opsi dalam bentuk object.
 await bot.api.sendMessage(12345, "Halo!", {/* opsi lainnya */});
+// Memeriksa object message dari pesan yang telah terkrim.
```
Comment (fadzikri) [inline site/docs/id/guide/basics.md:44]:
```suggestion
// Memeriksa object message dari pesan yang telah terkirim.
```

Comment (fadzikri) [reply site/docs/id/guide/basics.md:44]: Do we need `telah terkirim`? or just `terkirim`? It's past tense sentence, what about `dikirim`? How?

**Review by fadzikri — CHANGES_REQUESTED** [review]
There's typo and need changes

File: site/docs/id/guide/basics.md (hunk 2 above)
Comment (CikiMomogi) [reply site/docs/id/guide/basics.md:44]: Shouldn't it be in the past tense because the message is sent first and then print the response?

Comment (CikiMomogi) [reply site/docs/id/guide/basics.md:44]: I think all of them is right. Maybe we can use `terkirim` or `dikirim` to make it shorter.

Comment (fadzikri) [reply site/docs/id/guide/basics.md:44]: I updated that **suggestion** use `dikirim` word for preview. What do you think?

Comment (CikiMomogi) [reply site/docs/id/guide/basics.md:44]: Sounds good

**Comment by Nazar-Ant** [comment]
Do we, the Ukrainian translation, have to join this PR or is it enough for us to simply synchronize the changes without marking them here?

**Comment by roj1512** [comment]
Latter

**Comment by KnorpelSenf** [comment]
> Do we, the Ukrainian translation, have to join this PR or is it enough for us to simply synchronize the changes without marking them here?



> Latter

@deptyped @LWJerri this may be interesting to you, too

**Review by fadzikri — APPROVED** [review]

**Review by habemuscode — APPROVED** [review]

**Pull request merged**